components/Arrow/Arrow.tsx

```diff
@@ -1,20 +1,47 @@
-import css from 'styled-jsx/css'
-export default function Arrow(){
+export default function Arrow({arrowOrientation}){
+    let rotation = '0';
+    let viewBox = "-7 -3 70 70";
+    switch (arrowOrientation){
+        case 'top':
+            rotation = '-90';
+            viewBox = "-30 10 70 70";
+            break;
+        case 'right':
+            rotation = '0';
+            viewBox = "-7 -3 70 70";
+            break;
+        case 'bottom':
+            rotation = '90';
+            viewBox = "-33 -27 70 70";
+            break;    
+        case 'left':
+            rotation = '180';
+            viewBox = "-7 -3 70 70";
+            break;
+        default:
+            rotation = '0';
+            viewBox = "-7 -3 70 70";
```

Comment:
where does viewBox values come from and how does it work?

Reply:
This is the default property for svg. Sets "path" in a specific location within the svg. For each rotation it just has to be different so I put it in a switch